solidity/random-beacon/test/RandomBeacon.Relay.test.ts

```diff
@@ -233,12 +237,49 @@ describe("RandomBeacon - Relay", () => {
                         firstEligibleMemberIndex + 2,
                         blsData.groupSignature
                       )
+
+                    // Wait until tx is mined to get tx block timestamp.
+                    const receipt = await tx.wait()
+                    txTimestamp = (
+                      await ethers.provider.getBlock(receipt.blockNumber)
+                    ).timestamp
                   })
 
                   it("should remove members who did not submit from the sortition pool", async () => {
-                    await expect(tx)
-                      .to.emit(sortitionPool, "OperatorsRemoved")
-                      .withArgs([member16.address, member17.address])
+                    // Two members should be kicked out from the pool.
+                    expect(await sortitionPool.operatorsCount()).to.be.equal(
+                      constants.groupSize - 2
+                    )
+
+                    // Member 16 should be kicked out from the pool, blocked
+                    // against re-joining for 2 weeks, and his gas deposit
```

Comment:
> (...) blocked against re-joining for 2 weeks (...)

Is there an easy way to verify that? Can we test when this member tries to be reinserted back to the sortition pool before 2 weeks pass?

Reply:
It is already tested. See tests of `registerOperator` method.